asset/src/kafka_sender/schema.ts

```diff
@@ -1,14 +1,47 @@
-import { ConvictSchema } from '@terascope/job-components';
+import {
+    ConvictSchema, ValidatedJobConfig, getOpConfig, get
+} from '@terascope/job-components';
 import { KafkaSenderConfig } from './interfaces';
 
+function fetchConfig(job: ValidatedJobConfig) {
+    const opConfig = getOpConfig(job, 'kafka_sender');
+    if (opConfig == null) throw new Error('Could not find elasticsearch_bulk operation in jobConfig');
```

Comment:
If you are going to copy and paste, make sure to fix references

```suggestion
    if (opConfig == null) throw new Error('Could not find kafka_sender operation in jobConfig');
```